action.yml

```
@@ -1,26 +1,41 @@
 ﻿# action.yml
-name: 'Install Snowflake CLI'
-description: 'Download and install snowflake-cli through pipx'
+name: "Install Snowflake CLI"
+description: "Download and install snowflake-cli through uv"
 branding:
-  icon: 'terminal'
-  color: 'blue'
-  
+  icon: "terminal"
+  color: "blue"
+
 inputs:
   cli-version:
-    description: 'Snowflake CLI version (by default, installs the latest version)'
+    description: "Snowflake CLI version (by default, installs the latest version)"
     required: false
 
   default-config-file-path:
-    description: 'Path to the default config file'
+    description: "Path to the default config file"
     required: false
-    default: './config.toml'
+    default: "./config.toml"
 
   custom-github-ref:
-    description: 'GitHub branch, tag, or commit to install from (if set, installs from GitHub; otherwise, installs from PyPI)'
+    description: "GitHub branch, tag, or commit to install from (if set, installs from GitHub; otherwise, installs from PyPI)"
+    required: false
+
+  use-oidc:
+    description: "Enable OIDC authentication"
+    required: false
+    default: "false"
+
+  oidc-token-name:
+    description: "Name of the token environment variable for OIDC authentication"
```

This might be a useful option for some, but might be hard to guess how/when to use it. I'd expand the description:
```suggestion
    description: "Name of the token environment variable for OIDC authentication. By default it's set for [temporary connections](https://github.com/snowflakedb/snowflake-cli-action/pull/53/files#r2231644092), however it can be changed to SNOWFLAKE_CONNECTIONS_<NAME>_TOKEN to work with named connections."
```